Userland/Utilities/less.cpp

```diff
@@ -20,6 +20,10 @@ static Atomic<bool> g_resized { false };
 
 static Atomic<bool> g_restore_buffer_on_close { false };
 
+constexpr size_t line_number_column_min_padding = 7;
```

Comment:
I've set the minimum column width to 7 here to match the behavior of `less` on Linux and FreeBSD. Setting this value to 2 instead maximizes the horizontal space that can be used, but results in text being pushed to the right when we reach line 100. It looks like this:

https://github.com/SerenityOS/serenity/assets/2817754/98f63ddc-9628-4414-894a-67f5d3842d72

Alternatively, we could set the value somewhere in between these two :shrug: